src/mongoc/mongoc-topology-scanner-private.h

```diff
@@ -44,6 +44,7 @@ typedef struct mongoc_topology_scanner_node
    mongoc_async_cmd_t             *cmd;
    mongoc_stream_t                *stream;
    int64_t                         timestamp;
+   int64_t                         last_used;
```

Comment:
This is messy, but I'm not quite sure where else to hang it.  Maybe on the streams themselves?  Bleh